pkg/policies/policies.go

```diff
@@ -183,6 +192,70 @@ func (pv *PolicyVerifier) evaluatePolicyAttachment(ctx context.Context, attachme
 	}, nil
 }
 
+func (pv *PolicyVerifier) computeArguments(inputs []*v1.PolicyInput, args map[string]string, bindings map[string]string) (map[string]string, error) {
+	result := make(map[string]string)
+
+	// Policies without inputs in the spec
+	// TODO: Remove this in next release, once users have migrated their policies
+	if len(inputs) == 0 {
+		result = args
+	}
+
+	// Check for required inputs
+	for _, input := range inputs {
+		if _, ok := args[input.Name]; !ok {
+			if input.Required {
```

Comment:
I thought we discussed that required will work in tandem with default?

Reply:
Added the validation to either one or the other.